terracottabigmemorymax/helm/templates/server-configmap.yaml

```diff
@@ -62,9 +62,9 @@ data:
                 <user>user</user>
               </auth>
               <management>
-                <ia>https://{{ include "common.names.fullname" $ }}-tmc-service.{{ $.Release.namespace }}.svc.cluster.local:{{ $.Values.terracotta.tmcsecureport }}/tmc/api/assertIdentity</ia>
+                <ia>https://{{ include "common.names.fullname" $ }}-tmc-service.{{ $.Release.Namespace }}.svc.cluster.local:{{ $.Values.terracotta.tmcsecureport }}/tmc/api/assertIdentity</ia>
```

Comment:
tmc-0 need to be appended as statefulset name for tmc is tmc so the full name should be https://tmc-0.{{ include "common.names.fullname" $ }}-tmc-service.{{ $.Release.Namespace }}.svc.cluster.local